reformat_dbt_sap_readme_and_update_yml (#36)
fivetran/dbt_sap: CHANGELOG.md

# dbt_sap v0.3.1

[PR #36](https://github.com/fivetran/dbt_sap/pull/36) includes the following updates:

> ## Documentation ##

Reviewer: The header notation is only applied to the left side in markdown
```suggestion
## Documentation
```

> - Updated extrator_reports.yml column descriptions for debit_amount, credit_amount, accumulated_balance, and turnover fields in the sap_0fi_gl10 model
> - accumulated_balance and turnover are both derived from the same source field, "balance carried forward" in a specific currency and will be identical amounts in the end model, as well as matching either the credit_amount or debit_amount depending on the credit/debit designation in the source
> - This duplication is deliberate and protects both legacy systems and downstream analytics

Reviewer: Typically one primary bullet should only relate to a single type of change. So in this case I would recommend having a primary bullet to detail the description changes and then sub-bullets to provide further details if necessary. Additionally, when writing the CHANGELOG entry it's best to put focus on the model impact and make it easily understandable at a glance. Lastly, when referencing fields or models in the CHANGELOG we apply the code block notation to ensure they are easily visible when users are reviewing the updates. See below for suggested changes.
```suggestion
- Updated the following column descriptions within the [sap__0fi_gl10](https://fivetran.github.io/dbt_sap/#!/model/model.sap.sap__0fi_gl_10) model: `debit_amount`, `credit_amount`, `accumulated_balance`, `turnover`. 
  - `accumulated_balance` and `turnover` are both derived from the same source field, "balance carried forward" in a specific currency and will be identical amounts in the end model, as well as matching either the `credit_amount` or `debit_amount` depending on the credit/debit designation in the source.
  - This duplication is deliberate and protects both legacy systems and downstream analytics.
```

Reviewer: You mention,

> `accumulated_balance` and `turnover` are both derived from the same source field, "balance carried forward"

What is the original name of that source field? It will be helpful to list out the exact name of the source field as seen in the destination. It's helpful to include the detailed name, but it will also be helpful to include the original name when tracing back through the code.

This will probably help address my comment in the `extractor_reports.yml` file regarding how these fields all relate to one another when looking at the code.
